Sources/ContextPropagation/Instrumentation/MultiplexInstrumentationMiddleware.swift

```diff
@@ -0,0 +1,21 @@
+public struct MultiplexInstrumentationMiddleware<InjectInto, ExtractFrom> {
```

Comment:
Add some documentation?

You can base it on https://github.com/apple/swift-log/blob/master/Sources/Logging/Logging.swift#L459-L464

basically replacing the "logging" words with our domain :)